### ISSUE TITLE
Cannot have multiple g-cubes on a page

### ISSUE THREAD

**Issue by mikebolt** [molarmanful/gCube#3]
If you have two or more g-cubes on a single page, only one will show up. It should be possible to have multiple g-cubes on a single page, side by side, with each operating independently of the other and having its own settings tags.

**Comment by molarmanful** [comment]
There is a temporary workaround that utilizes a technique used by most cube applets these days:
1. Put your cube in a separate HTML file, and write up a script that reads URL parameters and converts them to gCube parameters.
2. Link to that HTML file using an iframe.

**Comment by mikebolt** [comment]
That's a valid workaround, but it won't work in many use cases, and it would require a bit of work. Hopefully I don't eat these words, but it looks like an easy fix that will just involve making the cube not rely on global properties.

**Comment by mikebolt** [comment]
I'm working on this in the 'multicube' branch. I just did a substantial refactor of the code. I got two of the cubes to work (well, show up) on one page but there are still some little issues. I'm going to keep working on it then merge it into master when it's ready.

**Comment by molarmanful** [comment]
Cool, thanks!

**Comment by mikebolt** [comment]
I believe that I have fixed the bugs in the cuber project that were preventing multiple cubes from working on the same page. The trickiest part was the 'time' variable, which caused problems because it was in a not-so-obviously static scope. I will clean up my work and commit this soon. Because I had to edit the compiled cuber code, I will just put my fixes in there for now, but in the future I am hoping to have a github project dedicated to maintaining and improving the cuber project. Then we could add it as a subrepo or something. I am also going to make a pull request in Tween.js, but it's not super vital that it gets accepted.
